UI/app/src/main/java/com/example/pathfinder/Activities/DriverDashboard.java

```diff
@@ -202,6 +204,7 @@ public void onCruiseControlBtn(View view) {
         } else {
             drive(IDLE_SPEED, STRAIGHT_ANGLE, "stopping");
         }
+        cruiseControl = view;
```

Comment:
Why do we need to declare a new view here to assign it later on the onClick method and not assign it to the already existing view via the ID ? 🤔

Reply:
> Why do we need to declare a new view here to assign it later on the onClick method and not assign it to the already existing view via the ID ? 🤔

Well I needed the view in order to use the performClick() and couldn't think of another way to do it.

Reply:
It would be something like:
In the onCreate() method :
ToggleButton cruiseControl = R.id.cruise_control

And where you need call
cruiseControl.performClick()
And then you don't need to assign the view in this method. (Which prevents the view to change if another view is using that method)
But you version works also, just concerned if at some point the onClick method for cruise control is used with other views it would create maybe conflicts

Reply:
@gusaxtcha and I agreed that I would make the changes presented above to allow him to work on another task.